### ISSUE TITLE
Are the releases up to date with Maven release?

### ISSUE THREAD

**Issue by jeffreynelson** [a-langer/eclipse-oomph-console#9]
How do we check for the latest release?

**Comment by a-langer** [comment]
Yes, the installer version matches the release in Maven, see https://search.maven.org/artifact/com.github.a-langer/org.eclipse.oomph.console.product. The latest version can be viewed at [CHANGELOG.md](https://github.com/a-langer/eclipse-oomph-console/blob/main/CHANGELOG.md). At the moment the latest version is `1.0.4`.

I also created a release page with download links for https://github.com/a-langer/eclipse-oomph-console/releases.

**Comment by a-langer** [comment]
To not miss the new version subscribe to updates: Watch -> Custom -> mark Releases -> Apply.
